posts/structural-search-patterns.md

```diff
@@ -0,0 +1,106 @@
+---
+title: How To Search with Sourcegraph using Structural Patterns
+author: marek-zaluski
+tags: [tutorial, search, Sourcegraph]
+description: Learn how to use structural search patterns to search code on Sourcegraph.
+image: /headers/sourcegraph-learn-header-8.svg
+imageAlt: Sourcegraph Learn
+socialImage: /headers/sourcegraph-learn-header-8.png
+type: posts
+---
+
+Structural search helps you search code for syntactical code patterns like function calls, arguments, `if`...`else` statements, and `try`...`catch` statements. It's useful for finding nested and recursive patterns as well as multi-line blocks of code.
+
+Structural search patterns are one of the three search patterns supported by Sourcegraph, along with literal patterns and regular expressions. They're different from regular expressions because they take into account the syntax of code, like balanced brackets, quoted strings, and delimiters.
+
+## Enabling structural search on Sourcegraph
+
+Enable structural search by clicking the icon of two square brackets (<CodeBracketsIcon/>) to the right of the search box:
+
+<GifLikeVideo url="/tutorial-images/enable-structural-search.mp4"/>
```

Comment:
I had to sit on the page for a bit to realize that this is animated (I knew based on it being an mp4), would it be possible to clip the video at the beginning so it is more obvious when someone is scrolling down?